modules/tinymce/src/plugins/anchor/main/ts/api/Commands.ts

```diff
@@ -6,8 +6,9 @@
  */
 
 import * as Dialog from '../ui/Dialog';
+import Editor from 'tinymce/core/api/Editor';
```

Comment:
This import is out of order. Won't matter while merging to master, but will fail when we try to merge to develop.